src/CryptoAxus.Application/Dto/BaseDto.cs

```diff
@@ -4,13 +4,26 @@ public abstract class BaseDto
 {
     public ObjectId? Id { get; set; }
 
-    public DateTime? CreatedDate { get; set; }
+    public DateTime? CreatedDate { get; set; } = DateTime.UtcNow;
 
-    public ObjectId? CreatedBy { get; set; }
+    public ObjectId? CreatedBy { get; set; } 
 
     public DateTime? LastModifiedDate { get; set; }
 
     public ObjectId? LastModifiedBy { get; set; }
 
-    public bool? IsDeleted { get; set; }
+    public bool? IsDeleted { get; set; } = false;
```

Comment:
No predefined values should be set for DTO's

Reply:
Resolved